library/ZendService/Apple/Apns/Message.php

```diff
@@ -256,6 +262,32 @@ public function setSound($sound)
     }
 
     /**
+     * Get Mutable Content
+     * 
+     * @return int|null
+     */
+    public function getMutableContent()
+    {
+        return $this->mutableContent;
+    }
+
+    /**
+     * Set Mutable Content
+     * 
+     * @param int|null $value
```

Comment:
@Necsord then we should currently validate the value to be in the range of `[null, 1]`, and reject other values